blog/models.py

```diff
@@ -17,6 +17,7 @@ class Blog(models.Model):
                              on_delete=models.CASCADE,
                              null=True, blank=True)
     status = models.CharField(max_length=1, choices=STATUS, default=0)
+    cover = models.ImageField(upload_to='images/')
```

Comment:
This field is already on `Image` so please drop it here, you'll need a foreign key there that links back to this post model